pages/index.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div>
     <HeroBanner @clickCTA="showModal" />
+    <TestimonialLists />
```

Comment:
this is wrong place component `testimonials`, this is should be above  component `mapsdedi`